README.md

```diff
@@ -1,46 +1,85 @@
+- YarnJs (v. 1.22.10)
+- Git
 
-### `yarn build`
+## Live Demo
 
-Builds the app for production to the `build` folder.<br />
-It correctly bundles React in production mode and optimizes the build for the best performance.
+Live demo for this project can be found in this [link](https://clever-booth-711741.netlify.app/).
 
-The build is minified and the filenames include the hashes.<br />
-Your app is ready to be deployed!
+## Getting Started
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+To run this project, you only need a computer with internet connection and a browser installed, and follow these steps:
 
-### `yarn eject`
+1. In your terminal, in the folder of your preference, type the following bash command to clone this repository:
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+```sh
+git clone git@github.com:mariordgez/covid-tracking-app.git
+```
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+2. Now that you have already cloned the repo run the following commands to get the project up and running:
 
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+```sh
+cd covid-tracking-app
+yarn
+yarn start
+```
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+This should start your local server in [http://localhost:3000/](http://localhost:3000/). To stop it, hit `<CTRL> + C` on your keyboard.
 
-## Learn More
+## Run tests
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+To test the App's components and logic, please, run the following command:
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+```sh
+yarn test --watchAll
+```
+
+When the tests are finished, just hit `q` key to leave the watch mode for the tests.
+
+## Author
+
+👤 **Mario Alberto Rodriguez Cota**
+
+- GitHub: [@mariordgez](https://github.com/mariordgez)
+- Twitter: [@MarioRo75396624](https://twitter.com/MarioRo75396624)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/mario-alberto-rodriguez-cota-a2860a205)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Emmanuelaaron/spaceships/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
```

Comment:
- [ ] Kindly give credit to Nelson Sakwa, Creator of the original design as per the project requirements.